doc/openshift-guide.md

```diff
@@ -64,43 +64,79 @@ You should now see a section **Production: Self-managed Gateway** at the bottom
-
-2. Start the OpenShift Origin VM
-  - Start a command prompt, create a new directory and change into it.
-  - At the terminal prompt run:
-    <pre><code>vagrant init openshift/origin-all-in-one</code></pre>
-   That will create a file called _Vagrantfile_ in the current directory.
-  - Edit the generated Vagrantfile. You will need to expose port `8080` of the gateway on your virtual machine, in order to do this you need to add the following line in the configuration:
-   <pre><code>config.vm.network "forwarded_port", guest: 8080, host: 8080, auto_correct: true</code></pre>
-  You can also change settings in Vagrantfile to give your virtual machine more or less resources (for example, you can adjust the the memory of the virtual machine with `vb.memory`)
-  - At the terminal prompt start the VM by typing:
-   <pre><code>vagrant up --provider=virtualbox</code></pre>
-  **Note**: Downloading the image and starting the VM may take several minutes (it may take a bit longer if you're running Windows).
-  <img src="https://support.3scale.net/images/screenshots/guides-openshift-vagrant-up-output.png" alt="Vagrant Output" >
-
- Upon starting the VM, you'll see  output which includes instructions for logging in to your OpenShift Origin VM, such as (don't log in yet):
-
- <pre><code>oc login https://10.2.2.2:8443</code></pre>
-
- Where _10.2.2.2_ is your _YOUR-OPENSHIFT-VM-IP_
-
- **Note**: Note down your _YOUR-OPENSHIFT-VM-IP_ as you'll need to use it repeatedly in this tutorial.
-
-3. Configure a host for the OpenShift Origin VM on your local machine. This will later allow you to expose the managed API using a hostname. This tutorial uses the hostname _gateway.openshift.demo_
- You'll need to add an entry for the Origin VM to your local hosts file.If your host PC is linux-based, this file is `/etc/hosts`
- If you are running Windows PC, this hosts file would typically be: `C:\Windows\System32\drivers\etc\hosts`
- If you are running Mac, you can type `$ nano /etc/hosts` in your terminal.
- Add a new line like this:
- 
- <pre><code>YOUR-OPENSHIFT-VM-IP gateway.openshift.demo</code></pre>
-
- Example:
- 
- <pre><code>10.2.2.2 gateway.openshift.demo</code></pre>
+There are many ways you can install Openshift.
+- [Running Openshift using Vagrant] (https://support.3scale.net/guides/infrastructure/docker-openshift)
+- [Running Openshift as all-in-one container] (https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) (current tutorial)
+
+## Installation
+The openshift cluster was installed and tested on 
+```
+CentOS version - CentOS Linux release 7.2.1511 (Core)
+Docker - 1.10.3
+Openshift Origin command line interface (CLI) - v1.3.1
+```
+
+1. Install Docker on CentOS
```

Comment:
The instructions work, but don't seem to be following this Docker documentation: https://docs.docker.com/engine/installation/linux/centos/